pyresample/test/test_utils.py

```diff
@@ -11,6 +12,16 @@ def tmp(f):
     return f
 
 
+def tmptiff(width=100, height=100, transform=None, crs=None, dtype=np.uint8):
+    import rasterio
+    array = np.ones((width, height)).astype(dtype)
+    fname = '/vsimem/%s' % uuid.uuid4()
```

Comment:
I understand that you want to create a unique, temporary filename, right ? In this case, I would suggest using the build tempfile library:
```suggestion
    fp, fname = mkstemp()
```

Reply:
and of course `from tempfile import mkstemp` on top of the file.

Reply:
It is not a simple temporary file, it is [in-memory](https://www.gdal.org/gdal_virtual_file_systems.html#gdal_virtual_file_systems_vsimem) file and we cannot use `tempfile` module for creating such files.

Reply:
Do I need to do something or we are waiting fix for rasterio from conda-forge community?

Reply:
I'll work on it. I think I had a workaround but with github's issues over the last 24 ours I haven't been able to test it.

Reply:
Ok, so `vsimem` is special path that gdal and rasterio understand to be in memory ? So I don't need to have a `/vismem` on my system ?

Reply:
Yes, exactly.

Reply:
Fine by me then.